### ISSUE TITLE
Typos in code

### ISSUE THREAD

**Issue by arkhodamoradi** [aa-samad/conv_snn#6]
line: https://github.com/aa-samad/conv_snn/blob/master/CIFAR10-DVS/dvscifar_dataloader.py#L117
should be:
`torch.save([torch.Tensor(frames), torch.Tensor([labels,])],`

line: https://github.com/aa-samad/conv_snn/blob/master/CIFAR10-DVS/main.py#L211
should be:
`output = self.model(data0.cuda())`

line: https://github.com/aa-samad/conv_snn/blob/master/CIFAR10-DVS/main.py#L217
should be:
`prec1, prec5 = accuracy(output.data, label.cuda(), topk=(1, 5))`

line: https://github.com/aa-samad/conv_snn/blob/master/CIFAR10-DVS/main.py#L227
should be:
`return top1.avg`

line: https://github.com/aa-samad/conv_snn/blob/master/CIFAR10-DVS/main.py#L133
should be:
`prec1 = self.validate_1epoch()`

I would also get rid of this line: https://github.com/aa-samad/conv_snn/blob/master/CIFAR10-DVS/main.py#L132

line: https://github.com/aa-samad/conv_snn/blob/master/CIFAR10-DVS/network.py#L165
should be:
`x = torch.cat(c2_spike[1::2], dim=1)`

**Comment by aa-samad** [comment]
Thank you for sharing your comments on the code.
